hack/boilerplate.go.txt

```diff
@@ -1,15 +1,3 @@
-/*
-Copyright 2020 The Crossplane Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
\ No newline at end of file
+// SPDX-FileCopyrightText: 2024 The Crossplane Authors <https://crossplane.io>
```

Comment:
did you already see where/how these headers may be used for build tool automation? i haven't seen that myself yet 🤔 i.e., is there more that would be needed besides this header update?

Reply:
I think I update this part according to the feedback from a [PR](https://github.com/crossplane/upjet-provider-template/pull/69#discussion_r1697112347) in provider-upjet-template. It seems related to code generator but not sure about it. 😶‍🌫️